firebase-firestore/CHANGELOG.md

```diff
@@ -7,8 +7,10 @@
 - [feature] Added `clearPersistence()`, which clears the persistent storage
   including pending writes and cached documents. This is intended to help
   write reliable tests (https://github.com/firebase/firebase-js-sdk/issues/449).
-- [changed] Firestore now provides a clear error message when it fails to load
-  SSL Ciphers during a connection attempt.
+- [changed] Instead of failing silently, Firestore now crahses the client app
+  if it fails to load SSL Ciphers. To avoid these crashes, you must bundle 
+  Conscrupt if you support non-GMSCore devices on Android KitKat or JellyBean 
```

Comment:
Another typo: Conscrypt. :)

Reply:
This was already fixed: https://github.com/firebase/firebase-android-sdk/pull/508/commits/080792b2646c9b070cc2dbcfe9f5c3c7f638119f